web/modules/custom/solution/src/EventSubscriber/WorkflowTransitionEventSubscriber.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\solution\EventSubscriber;
+
+use Drupal\message\Entity\Message;
+use Drupal\message\Entity\MessageTemplate;
+use Drupal\message_notify\MessageNotifier;
+use Drupal\og\Entity\OgMembership;
+use Drupal\state_machine\Event\StateChangeEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Event subscribers for Organic Groups.
+ */
+class WorkflowTransitionEventSubscriber implements EventSubscriberInterface {
```

Comment:
If I'm not mistaking, @sandervd  and @pfrenssen  where against this idea of using the Rules module so I would not go with it at least for now.